src/parse.yy

```diff
@@ -1667,6 +1681,25 @@ initializer_list
       }
     ;
 
+attribute_list
```

Comment:
I think ``attribute_list`` is now misleading. In this definition it's not a list. It's a single node.
Just "pragma" or "single_pragma" might be a better name.